### PR TITLE
Update/docs faq ascat

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: Reference section to docs/FAQ.rst

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: docs/FAQs.rst [hunk 1]
```diff
@@ -96,3 +96,53 @@ Make a pull request to master at this point. After pull request is approved and
...
+**References**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**How to generate reference files for ascatNGS**
+
+Detailed information is available at the [ascatNGS](https://github.com/cancerit/ascatNgs)
+
+Briefly, ascatNGS needs gender loci file if gender information for the imput sample is not available. Second file is SnpGcCorrections.tsv, which is prepared from 1000 genome SNP panel.
+1. Gender loci file:
+ - GRCh37d5_Y.loci contains the following contents:
```
Comment (hassanfa) [inline docs/FAQs.rst:110]:
```suggestion
1. Gender loci file:

GRCh37d5_Y.loci contains the following contents:
```

File: docs/FAQs.rst [hunk 2]
```diff
@@ -96,3 +96,53 @@ Make a pull request to master at this point. After pull request is approved and
...
+
+```
+Y       4546684
+Y       2934912
+Y       4550107
+Y       4549638
+```
+
+2. GC correction file:
+ - First step is to download 1000 genome snp file and convert it from .vcf to .tsv. The detailed procedure to for this step is provided [here](https://github.com/cancerit/ascatNgs/wiki/Human-reference-files-from-1000-genomes-VCFs).
```
Comment (hassanfa) [inline docs/FAQs.rst:120]:
```suggestion
2. GC correction file:

First step is to download 1000 genome snp file and convert it from .vcf to .tsv. The detailed procedure to for this step is provided [here](https://github.com/cancerit/ascatNgs/wiki/Human-reference-files-from-1000-genomes-VCFs).
```

File: docs/FAQs.rst [hunk 3]
```diff
@@ -96,3 +96,53 @@ Make a pull request to master at this point. After pull request is approved and
...
+--or--
+
+```console
+curl -sSL $TG_DATA | zgrep -F 'E_Multiple_observations' | grep -F 'TSA=SNV' |\
+ perl -ane 'next if($F[0] !~ m/^\d+$/ && $F[0] !~ m/^[XY]$/); $F[7]=~m/MAF=([^;]+)/; next if($1 < 0.05); next if($F[0] eq $l_c && $F[1]-1000 < $l_p); printf "%s\t%s\t%d\n", $F[2],$F[0],$F[1]; $l_c=$F[0]; $l_p=$F[1];' \
+> SnpPositions_GRCh37_1000g.tsv
+```
+
+
+ - Second step is to use SnpPositions.tsv file and generate SnpGcCorrections.tsv file as described [here](https://github.com/cancerit/ascatNgs/wiki/Convert-SnpPositions.tsv-to-SnpGcCorrections.tsv).
```
Comment (hassanfa) [inline docs/FAQs.rst:144]:
```suggestion
 Second step is to use SnpPositions.tsv file and generate SnpGcCorrections.tsv file as described [here](https://github.com/cancerit/ascatNgs/wiki/Convert-SnpPositions.tsv-to-SnpGcCorrections.tsv).
```

**Review by hassanfa — CHANGES_REQUESTED** [review]
codeblocks are not working properly:

image

To test documentation locally, you can try to build it locally and check if it working properly:

https://github.com/Clinical-Genomics/BALSAMIC/blob/master/docs/README.rst

**Comment by hassanfa** [comment]
You can consult https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html for how to format rst.

File: docs/FAQs.rst (hunk 3 above)
Comment (khurrammaqbool) [reply docs/FAQs.rst]: @hassanfa I mistakenly added markdown codes. now fixed!

File: docs/FAQs.rst [hunk 4]
```diff
@@ -96,3 +96,65 @@ Make a pull request to master at this point. After pull request is approved and
...
+**References**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**How to generate reference files for ascatNGS**
+
+Detailed information is available at the ascatNGS_
+
+.. _ascatNGS: https://github.com/cancerit/ascatNgs
+
+Briefly, ascatNGS needs gender loci file if gender information for the imput sample is not available. Second file is SnpGcCorrections.tsv, which is prepared from 1000 genome SNP panel.
```
Comment (ashwini06) [inline docs/FAQs.rst:110]:
```suggestion
Briefly, ascatNGS needs a gender loci file if gender information for the input sample is not available. The second file is `SnpGcCorrections.tsv`, which is prepared from the 1000 genome SNP panel.
```

File: docs/FAQs.rst [hunk 5]
```diff
@@ -96,3 +96,65 @@ Make a pull request to master at this point. After pull request is approved and
...
+1. Gender loci file:
```
Comment (ashwini06) [inline docs/FAQs.rst:111]:
```suggestion

1. **Gender loci file:**
```

File: docs/FAQs.rst [hunk 6]
```diff
@@ -96,3 +96,65 @@ Make a pull request to master at this point. After pull request is approved and
...
+  
+GRCh37d5_Y.loci contains the following contents:
+
+.. line-block::
+    Y	4546684
+    Y	2934912
+    Y	4550107
+    Y	4549638
+
+2. GC correction file:
```
Comment (ashwini06) [inline docs/FAQs.rst:121]:
```suggestion
2. **GC correction file:**
```

File: docs/FAQs.rst [hunk 7]
```diff
@@ -96,3 +96,65 @@ Make a pull request to master at this point. After pull request is approved and
...
+
+.. code:: console
+
+    curl -sSL $TG_DATA | zgrep -F 'E_Multiple_observations' | grep -F 'TSA=SNV' |\
+    perl -ane 'next if($F[0] !~ m/^\d+$/ && $F[0] !~ m/^[XY]$/); $F[7]=~m/MAF=([^;]+)/;\
+    next if($1 < 0.05); next if($F[0] eq $l_c && $F[1]-1000 < $l_p);\
+    printf "%s\t%s\t%d\n", $F[2],$F[0],$F[1]; $l_c=$F[0]; $l_p=$F[1];'\
+    > SnpPositions_GRCh37_1000g.tsv
+
+Second step is to use SnpPositions.tsv file and generate SnpGcCorrections.tsv file as described here_
```
Comment (ashwini06) [inline docs/FAQs.rst:153]:
```suggestion
Second step is to use `SnpPositions.tsv` file and generate `SnpGcCorrections.tsv` file as described here_
```

**Review by ashwini06 — COMMENTED** [review]
@khurrammaqbool:  I have few comments on markdown syntax as well as spellchecks/grammar. Try to use `highlighters` for certain words which give a better reading of the documentation

**Pull request merged**